### PR TITLE
Proper message display when no bash, python and YAML files changed in the PR

### PR DESCRIPTION
**Pull request by hansinikarunarathne**
# Pull Request Template for Kubeflow manifests Issues

## ✅ Contributor checklist
  - Make sure you have tested with kustomize. See [Installation Prerequisites](https://github.com/kubeflow/manifests#prerequisites)
  - All the commits have been _signed-off_  (To pass the `DCO` check)
  - Submit the [Contributor License Agreements](https://cla.developers.google.com/clas) (To pass the `cla/google` check)

---     
 
> You can join the CNCF Slack and access our meetings at the [Kubeflow Community](https://www.kubeflow.org/docs/about/community/) website. Our channel on the CNCF Slack is here [**#kubeflow-platform**](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2).

### CODE REVIEW

**Comment by juliusvonkohout** [comment]
You have to fix the DCO before i can merge.

File: apps/jupyter/notebook-controller/upstream/base/kustomization.yaml
Comment (juliusvonkohout) [inline apps/jupyter/notebook-controller/upstream/base/kustomization.yaml:1]: Why are you changing this file?

File: README.md
Comment (juliusvonkohout) [inline README.md:1]: Why are you changing this file?

Comment (hansinikarunarathne) [reply README.md:1]: The difference is showed in the same line. But I did not change it.

**Comment by juliusvonkohout** [comment]
/lgtm
/approve

we need to see it in action

**Pull request merged**